docs/changehistory/2.0.0.md

```diff
@@ -289,7 +289,7 @@ If you were using or extending the tslint configuration supplied by iModel.js, p
 The former `@bentley/imodeljs-clients` package contained clients for various Bentley-hosted services. It has now been split into the following packages under the `/clients/` directory:
 
 - `@bentley/itwin-client`
-- `@bentley/context-registry-client`
+- `@bentley/itwin-registry-client`

```

Comment:
Should we really change this? This is a `2.0.0` change notes and the package was named `@bentley/context-registry-client` at the time, so I believe it should be kept the way it was.

Reply:
I agree, it should stay as it was at the time.

Reply:
reverted the change.